linear_axis: 1
linear_scale: 0.2

rotation_axis: 0
rotation_scale: 1.0
